multihost/db-postgresql-with-custom-certificates/test.sh

```diff
@@ -67,6 +67,10 @@ function get_IP() {
 
 Attestation_server() {
     rlPhaseStartSetup "Attestation server setup"
+        rpm -q keylime-verifier 2> /dev/null && INSTALL_PKGS="$INSTALL_PKGS keylime-verifier"
+        rpm -q keylime-registrar 2> /dev/null && INSTALL_PKGS="$INSTALL_PKGS keylime-registrar"
+        [ -n "$INSTALL_PKGS" ] && rlRun "dnf --noautoremove -y remove $INSTALL_PKGS"
+
```

Comment:
I would also delete /etc/keylime and /var/lib/keylime.

Reply:
I believe tenant is configured later, also certificates are generated during the test. Therefore it should be safe to delete it at the beginning of the test, eventually after doing a backup (and later restore).